test/cli/testvarious

```diff
@@ -862,6 +862,29 @@ $EXE -f '%.7f' \
  -E >> ${OUT} <<EOF
 7461300.0 528000.0 0.0
 EOF
+##############################################################################
+echo  "##############################################################" >> ${OUT}
+echo  "Test genericsom forward projection" >> ${OUT}
+#
+$EXE -f '%.7f' \
+    +proj=latlong +datum=WGS84 +to \
+    +proj=genericsom +inc_angle=1.7157253262878522 +ps_rev=0.06866666666666667 \
+    +asc_lon=1.1242171183417042  +ellps=WGS84 +units=m \
+ -E >> ${OUT} <<EOF
+-120.001014 38.999576 0.
+EOF
+##############################################################################
+echo  "##############################################################" >> ${OUT}
+echo  "Test genericsom inverse projection" >> ${OUT}
+#
+$EXE -f '%.7f' \
+    +proj=genericsom +inc_angle=1.7157253262878522 +ps_rev=0.06866666666666667 \
+    +asc_lon=1.1242171183417042 +ellps=WGS84 +units=m +to +proj=latlong \
+    +datum=WGS84 -f '%0.7f' \
+ -E >> ${OUT} <<EOF
+15758237.5060846 427373.6032512 0.0
+EOF
+##############################################################################
```

Comment:
I would drop those changes. Testing with builtins.gie is enough and more portable (testvarious is a shell script and can't run on Windows)

```suggestion
```

Reply:
It's okay with me. I was matching misrsom behavior here though, so perhaps those tests should be removed as well for consistency?

Reply:
yes, likely.
one thing to be aware is that the tests in builtins.gie have been all generated for the same input coordinates, which don't necessarily tests all the relevant code paths of the given projection method. So sometimes it can be appropriate to insert input coordinates that are dependent on the projection method.

Reply:
Removed as requested